src/components/input/Select.tsx

```diff
@@ -0,0 +1,55 @@
+import classnames from 'classnames';
+import type { ComponentChildren, JSX } from 'preact';
+
+import type { PresentationalProps } from '../../types';
+
+import InputRoot from './InputRoot';
+
+type ComponentProps = {
+  children?: ComponentChildren;
+  hasError?: boolean;
+};
+export type SelectProps = PresentationalProps &
+  ComponentProps &
+  JSX.HTMLAttributes<HTMLSelectElement>;
+
+// URI-encoded source of `CaretDownIcon`
+// Currently, the color (stroke) is hard-coded
+const arrowImage = `url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='16' height='16' viewBox='0 0 16 16' aria-hidden='true' focusable='false'%3E%3Cg fill-rule='evenodd'%3E%3Crect fill='none' stroke='none' x='0' y='0' width='16' height='16'%3E%3C/rect%3E%3Cpath fill='none' stroke='%239c9c9c' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' d='M12 6l-4 4-4-4'%3E%3C/path%3E%3C/g%3E%3C/svg%3E")`;
```

Comment:
Alternately, we can extend the Tailwind theme in the package's preset:

```
theme: {
  extend: {
    backgroundImage: {
      'caret-down': 'url("/* ... *./")',
    }
  }
}
```

which could then be applied with the `bg-caret-down` utility class.

Putting the source of the image here in the component module means we could ostensibly set `stroke` color or other attribute values dynamically. The up side of putting it in the tailwind preset is that, ostensibly, a consumer could override it with something completely different.

Reply:
I guess there are pros and cons to both approaches, but since it's hard to see how the needs for this component will evolve, I would suggest keeping it like this and change it later if wee see the need.